components/tikv_util/src/config.rs

```diff
@@ -689,6 +689,88 @@ pub fn check_max_open_fds(expect: u64) -> Result<(), ConfigError> {
     }
 }
 
+#[macro_export]
+macro_rules! rocksdb_enum_mod {
```

Comment:
rename to numeric_enum_serializing_mod

Reply:
OK，The name that contains 'rocksdb' is indeed improper as this is a common lib